packages/network/connman/package.mk

```diff
@@ -18,7 +18,7 @@
 
 PKG_NAME="connman"
 # DO NOT UPGRADE!!
```

Comment:
i would keep it, we can update for prereleases, but we should keep the comment to see we must be carefully for releases